### PR TITLE
[Quest] Convert Beastmaster AF1 to IF

### PR DESCRIPTION
**Pull request by jamesbradleym**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Converts Beast AF1 Quest (Wings of Gold) to IF

## Steps to test these changes

Complete Beast AF1 Quest

### CODE REVIEW

File: scripts/quests/jeuno/Wings_of_Gold.lua [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+        [xi.zone.UPPER_JEUNO] =
+        {
+            ['Brutus'] =
+            {
+                onTrigger = function(player, npc)
+                    if not player:hasKeyItem(xi.ki.GUIDING_BELL) then
+                        return quest:event(136)
+                    else
+                        return quest:progressEvent(138) -- Finish Quest "Wings of Gold"
```
Comment (claywar) [inline scripts/quests/jeuno/Wings_of_Gold.lua]: This comment can most likely be removed

File: scripts/quests/jeuno/Wings_of_Gold.lua [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                player:getMainLvl() >= xi.settings.main.AF1_QUEST_LEVEL and
+                player:getMainJob() == xi.job.BST
+        end,
+
+        [xi.zone.UPPER_JEUNO] =
+        {
+            ['Brutus'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 1 then
```
Comment (claywar) [inline scripts/quests/jeuno/Wings_of_Gold.lua]: For clarity, I would suggest starting with a 0-value check, and reordering the condition.  That should most likely reduce the need for two in-line comments

File: scripts/quests/jeuno/Wings_of_Gold.lua (hunk 1 above)
Comment (jamesbradleym) [reply scripts/quests/jeuno/Wings_of_Gold.lua]: 🗑️

File: scripts/quests/jeuno/Wings_of_Gold.lua (hunk 2 above)
Comment (jamesbradleym) [reply scripts/quests/jeuno/Wings_of_Gold.lua]: ♻

**Comment by jamesbradleym** [comment]
Closing this PR to reduce open PRs and prevent staleness—will revisit.

**Pull request closed without merging**